### PR TITLE
feat: 1689 web validator app consumes execution resultsjson

### LINKED ISSUE #1689: Web validator app consumes execution_results.json
# Description

After #1688, 

- [x] consume the execution_results.json on the web app to notify the client when the validation process is complete with or without errors.
- [ ] cypress tests

### PR DESCRIPTION
**Pull request by qcdyx**
**Summary:**
Closes #1689 
consumes the results.json file.

**Expected behavior:** 
This code consumes the result.json file.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `gradle test` to make sure you didn't break anything
- [ ] Add or update any needed [documentation](https://github.com/MobilityData/gtfs-validator/tree/master/docs) to the repo 
- [x] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [x] Linked all relevant issues
- [ ] Include screenshot(s) showing how this pull request works and fixes the issue(s)

### CODE REVIEW

**Comment by qcdyx** [comment]
@davidgamez I would like to get your comments before you go on vacation.

File: .github/workflows/stg_web_svc_merge.yml [hunk 1]
```diff
@@ -10,17 +10,15 @@ env:
   NODE_VERSION: "20"
 
 jobs:
-
   web_cd:
     name: Deploy web service
     runs-on: ubuntu-latest
 
     env:
-      ENV_NAME              : stg
```
Comment (davidgamez) [inline .github/workflows/stg_web_svc_merge.yml:19]: I don't see any relevant changes to the issue; it's only formatting. Can you revert these changes?

File: web/client/src/routes/+page.svelte [hunk 2]
```diff
@@ -296,6 +297,35 @@
     });
   }
 
+  function reportExecutionFinished() {
```
Comment (davidgamez) [inline web/client/src/routes/+page.svelte:300]: This function looks the same as `reportExists` with only a URL as a difference. We can reuse `reportExists` by renaming it with a clear name, something like _remoteFileExists_, and adding a URL parameter.

File: .github/workflows/stg_web_svc_merge.yml (hunk 1 above)
Comment (qcdyx) [reply .github/workflows/stg_web_svc_merge.yml:19]: yes

Comment (qcdyx) [reply .github/workflows/stg_web_svc_merge.yml:19]: As long as I hit cmd+s in VSCode, it's reformatted automatically. I'm not sure how to resolve it.

**Comment by davidgamez** [comment]
I am closing this PR as I had issues after a master rebase. I created a twin PR #1712.
